### ISSUE TITLE
ability to flip maps in display

### ISSUE THREAD

**Issue by adf-ncgr** [ncgr/cmap-js#38]
this is a feature of the old cmap and of cmtv; the need for it is illustrated by the example map comparison we've put into the default config (Pv06 vs Vu06, in which the totally arbitrary choice of 
which end to call 0 in the linkage map results in an inverted view).

**Comment by awilkey** [comment]
Has been folded into feature/track-side changes

**Comment by Cheehau0659** [comment]
@adf-ncgr Could you verify this is fix in the master branch ?

**Comment by adf-ncgr** [comment]
Hi @Cheehau0659 - sure, I will try to get myself re-oriented to what this issue was about; however, it might take me a while to be able to provide verification so please be patient (but feel free to ping me about it)

**Comment by nathanweeks** [comment]
@adf-ncgr In case it helps, the demo (https://legumefederation.github.io/cmap-js) mentioned in the README is built from the master branch via GitHub Actions workflow.

**Comment by adf-ncgr** [comment]
yes that's super-helpful. It looks like you can do the flip by editing the json config and toggle "invert" properties there. So I guess the question would be if the UI is sufficient for the target userbase. works for me, though.

**Comment by nathanweeks** [comment]
I might be misunderstanding this feature. While it looks like the feature correspondence lines move when I click "Configuration", change an "invert" property from false to true (or vice versa), and select "Use New Configuration", I don't see the backbone/markers/QTL flip?

**Comment by adf-ncgr** [comment]
yes, you're right as usual, I was just looking at the behavior of the correspondence lines. 

Another bad behavior seemingly related to being in the inverted state is that zooming by drag-selecting on the left-hand representation of the map (light blue bar) puts you at the inverse location of where you thought you'd asked to go- e.g. drag-select a region at the top and you end up at the bottom. Note that in the demo the left-most map begins in the inverted state and shows this odd behavior initially, but tweaking the config to set inverted to false does not seem to fix the behavior either.
